integration/cri-o/crio_skip_tests.sh

```diff
@@ -27,6 +65,94 @@ declare -A skipCRIOTests=(
 ['test "annotations passed through"']='This is not working'
 ['test "ctr with default_env set in configuration"']='This is not working'
 ['test "ctr with absent mount that should be rejected"']='This is not working'
+['test "pids limit"']='cgroups.bats case is not working'
+['test "conmon custom cgroup"']='cgroups.bats case is not working'
```

Comment:
For this one, this is not a bug.
cri-o uses conmon with runc, but it doesn't use it with kata, so there is no reason to test it.
I would modify the comment as follow, so that we document why we skip it, and then we don't need to track anything more.

```suggestion
['test "conmon custom cgroup"']='conmon is not involved with kata runtime'
```